clang/lib/Analysis/UnsafeBufferUsage.cpp

```diff
@@ -366,6 +366,7 @@ isInUnspecifiedUntypedContext(internal::Matcher<Stmt> InnerMatcher) {
 
 namespace {
 
+/* TO_UPSTREAM(BoundsSafetyInterop) ON */
```

Comment:
I assume there is no problem in upstreaming the part of the interop in UnsafeBufferUsage.cpp?   @patrykstefanski